pkg/preparer/orchestrate.go

```diff
@@ -203,6 +219,11 @@ func (p *Preparer) verifySignature(manifest pods.PodManifest, logger logging.Log
 func (p *Preparer) installAndLaunchPod(newManifest *pods.PodManifest, pod Pod, logger logging.Logger) bool {
 	// do not remove the logger argument, it's not the same as p.Logger
 
+	if p.podIdForbidden(newManifest.ID()) {
```

Comment:
personally I'd rather have one nooping goroutine floating around than putting this logic in a place that's difficult to test. installAndLaunchPod seems like the better location to me